### ISSUE TITLE
udev.py deprecation warnings spamming the logs

### ISSUE THREAD

**Issue by M4rtinK** [storaged-project/blivet#533]
The Anaconda log is being spammed by deprecation warnings that originate mostly from the Blivet udev.py module:

```
13:37:44,149 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:174: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  name = udev_info["DM_NAME"]

13:37:44,149 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/populator/populator.py:282: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  log_method_call(self, name=name, info=pprint.pformat(dict(info)))

13:37:44,167 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/populator/populator.py:121: DeprecationWarning: device_is_realdisk will be removed in a future version. udev.device_is_disk provides same functionality in 3.0
  return udev.device_is_realdisk(info)

13:37:44,296 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:595: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  return self[key]

13:37:44,297 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/populator/populator.py:282: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  log_method_call(self, name=name, info=pprint.pformat(dict(info)))

13:37:44,316 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/populator/populator.py:121: DeprecationWarning: device_is_realdisk will be removed in a future version. udev.device_is_disk provides same functionality in 3.0
  return udev.device_is_realdisk(info)

13:37:44,328 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:174: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  name = udev_info["DM_NAME"]

13:37:44,400 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:595: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  return self[key]

13:37:44,400 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:174: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  name = udev_info["DM_NAME"]

13:37:44,401 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/populator/populator.py:282: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  log_method_call(self, name=name, info=pprint.pformat(dict(info)))

13:37:44,423 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/populator/populator.py:121: DeprecationWarning: device_is_realdisk will be removed in a future version. udev.device_is_disk provides same functionality in 3.0
  return udev.device_is_realdisk(info)

13:37:44,472 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:595: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  return self[key]

13:37:44,472 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:174: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  name = udev_info["DM_NAME"]

13:37:44,472 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/populator/populator.py:282: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  log_method_call(self, name=name, info=pprint.pformat(dict(info)))

13:37:44,489 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/populator/populator.py:121: DeprecationWarning: device_is_realdisk will be removed in a future version. udev.device_is_disk provides same functionality in 3.0
  return udev.device_is_realdisk(info)

13:37:44,531 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:44,534 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:174: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  name = udev_info["DM_NAME"]

13:37:44,570 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:44,572 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:174: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  name = udev_info["DM_NAME"]

13:37:44,595 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:44,596 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:44,620 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:44,621 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:44,746 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:44,746 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:44,771 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:44,772 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:44,849 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:44,850 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:44,874 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:44,875 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:44,898 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:44,899 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:44,924 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:44,925 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:44,966 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:44,967 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:44,992 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:44,993 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,074 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,075 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,100 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,101 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,125 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,126 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,151 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,152 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,195 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,197 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:174: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  name = udev_info["DM_NAME"]

13:37:45,222 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,224 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:174: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  name = udev_info["DM_NAME"]

13:37:45,344 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,345 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,369 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,370 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,393 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,394 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,420 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,420 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,485 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,486 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,510 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,511 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,534 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,534 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,559 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,560 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,624 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,625 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,649 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,650 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,674 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,675 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,699 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,700 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:117: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  if ret:

13:37:45,763 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,765 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:174: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  name = udev_info["DM_NAME"]

13:37:45,791 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,792 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:174: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  name = udev_info["DM_NAME"]

13:37:45,816 WARN anaconda: /usr/lib64/python3.5/_collections_abc.py:601: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  self[key]

13:37:45,818 WARN anaconda: /usr/lib/python3.5/site-packages/blivet/udev.py:174: DeprecationWarning: Will be removed in 1.0. Access properties with Device.properties.
  name = udev_info["DM_NAME"]

```

It would be really good if we can get rid of this spam as it's making the log files almost unreadable.

**Comment by vpodzime** [comment]
@vojtechtrefny do we have a solution for this now?

**Comment by dwlehman** [comment]
I believe this was resolved by #585.
